okta/resource_okta_group_memberships.go

```diff
@@ -66,20 +68,16 @@ func resourceGroupMembershipsCreate(ctx context.Context, d *schema.ResourceData,
 }
 
 func resourceGroupMembershipsRead(ctx context.Context, d *schema.ResourceData, m interface{}) diag.Diagnostics {
-	groupId := d.Get("group_id").(string)
-	users := convertInterfaceToStringSetNullable(d.Get("users"))
-	client := getOktaClientFromMetadata(m)
-	ok, err := checkIfGroupHasUsers(ctx, client, groupId, users)
+	var diags diag.Diagnostics
```

Comment:
We can punt have a `diags` variable defined here as it isn't used anywhere in the method other than the return statement. Instead, just return nil as the final statement. `diag.Diagnostics` is a type defined as a slice of `diag.Diagnostic` so `return nil` is equivalent to `return []diag.Diagnostic{}` at runtime.